doc/radosgw/s3/commons.rst

```diff
@@ -7,19 +7,19 @@
 
 Bucket and Host Name
 --------------------
-There are two different modes of accessing buckets. The first method identifies
-the bucket as the top-level directory in the URI::
+There are two different modes of accessing buckets: path-style and virtual-hosted-style.
+Path-style requests identify the bucket as the top-level directory of the request's path::
 
 	GET /mybucket HTTP/1.1
 	Host: cname.domain.com
 
-Most S3 clients nowadays rely on vhost-style access. The desired bucket is
-indicated by a DNS FQDN. For example::
+Most S3 clients default to virtual-hosted-style access, where the bucket name is instead
```

Comment:
I don't agree with "instead" being in this sentence, since it does not refer to anything within the grammatical scope of the sentence, but I'll deal with that in a future PR.